jobrunner/config.py

```diff
@@ -43,7 +43,10 @@ class ConfigException(Exception):
 
 BACKEND = os.environ.get("BACKEND", "expectations")
 
-USING_DUMMY_DATA_BACKEND = BACKEND == "expectations"
+if "USING_DUMMY_DATA_BACKEND" in os.environ:
```

Comment:
Just in passing, I generally like to check the env var is at least non-empty, rather than just present. If nothing else it makes it easier to override in development by writing `MYVAR= ./my-executable`.

But I think it's probably better to check the value as well because there's less potential for error e.g.
```py
USING_DUMMY_DATA_BACKEND = {"true": True, "false": False}[
    os.environ.get("USING_DUMMY_DATA_BACKEND", "false").strip()
]
```

Reply:
Ok, have done so, but with version that honours Postel's law, which I generally prefer for env var boolean flags

Reply:
Ah yeah, good shout